### DISCUSSION TITLE
Cannot add more Sponsor Tiers

### DISCUSSION THREAD

**Discussion by tomalaforge** [community/community#75962] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

We can normally add up to 10 sponsor tiers, and I have only 1 for the moment.

However when I try to add a new one, the Publish Tier or Save draft button stay disable even if the amount and description are filled up. (See picture below)

Can someone help me? Thanks

![image](https://github.com/community/community/assets/30832608/952a7a38-504a-4b9e-b66f-b4df2b2148fc)

**Comment by anjuan** [comment]
Hi @tomalaforge, I'm the Engineering Manager for GitHub Sponsors. I'm sorry that the tier editor isn't working as expected. On the tier editor, there's a line that says something like "You have # published tiers and # draft tiers. You can publish up to # more monthly tiers."  What does this line say for your profile?

**Reply by tomalaforge** [reply]
Thanks @anjuan for answering my issue. 
The line says that I can publish 9 other tiers (see image below)
<img width="746" alt="image" src="https://github.com/community/community/assets/30832608/dcd2273a-64ec-4160-a03a-237d23085b19">

On the screen to add a new tier, when I tap a number inside the amount input, I have a the request `https://github.com/sponsors/tomalaforge/dashboard/tier_verify?frequency=recurring` that is failing with a block status
<img width="536" alt="image" src="https://github.com/community/community/assets/30832608/e058f91e-d796-47e0-b29b-e0becd5b9411">


Still on the screen to add a new tier, when I disable the button manually (in the DOM), I have a verifying overlay that pops on the amount input, and disappear after a brief time. Without any http request or error in the console. 
<img width="183" alt="image" src="https://github.com/community/community/assets/30832608/78b670df-d323-4440-95c2-0718362eb3e2">

**Reply by mjimenez98** [reply]
Hello @tomalaforge 👋 I am part of the GitHub Sponsors team and I am eager to make this issue go away!

So far I haven't been able to reproduce this using Edge and Safari. Would it be possible to know which browser you are using and if you have any extensions like ad-blockers installed?

**Reply by tomalaforge** [reply, marked as answer]
I'm on Chrome and this is indeed the ad blockers which is blocking the button. I would have never think of that. 

Thanks a lot for your help 🙏

It should be written somewhere inside the documentation. 😅

**Reply by mjimenez98** [reply]
I am glad to hear we found the problem 🎉 

I agree with you about documenting this behavior so I have opened an issue in our internal project board with your suggestion! Thank you!

**Reply by anjuan** [reply]
@tomalaforge If you feel we've resolved this for you, do you mind marking this as answered?

**Comment by tomalaforge** [comment]
Resolved, ad blocker was blocking the request
